### PR TITLE
netsuite partial fetch documentation

### PR DESCRIPTION
**Pull request by tomermevorach**

### CODE REVIEW

File: packages/netsuite-adapter/specific-cli-options.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# NetSuite Specific CLI options
+## Partial Fetch
+Salto CLI allows the user to fetch only specific netsuite configuration elements and/or file cabinet items by appending the below CLI parameters following the ```-C``` CLI option:
+
+| Name                           | Default when undefined  | Description
+| -------------------------------| ------------------------| -----------
+| netsuite.fetchTarget.filePaths                          | fetch all cabinet files                     | A list of regexes of cabinet file paths. Any file whose path matches any of the regexes will be fetched
```
Comment (omrilit) [inline packages/netsuite-adapter/specific-cli-options.md:7]: regexes -> regular expressions?

**Review by omrilit — COMMENTED** [review]
Is that correct that when you state only `netsuite.fetchTarget.types.[type_name]` you don't **all** fetch the files as well?
It's written in that if you don't provide `netsuite.fetchTarget.filePaths` it fetches all file cabinet content, while in the examples it seems like it doesn't.
I think that some explanation is missing.

**Comment by alonstern** [comment]
> Is that correct that when you state only `netsuite.fetchTarget.types.[type_name]` you don't **all** fetch the files as well?
> It's written in that if you don't provide `netsuite.fetchTarget.filePaths` it fetches all file cabinet content, while in the examples it seems like it doesn't.
> I think that some explanation is missing.

That's true. The default of `types` and` file Paths` is to fetch nothing, but the default of the whole `fetchTarget` is to fetch everything. Might be a bit trickier to explain.

File: packages/netsuite-adapter/specific-cli-options.md (hunk 1 above)
Comment (tomermevorach) [reply packages/netsuite-adapter/specific-cli-options.md]: fixed

**Comment by tomermevorach** [comment]
> explanation

**Comment by tomermevorach** [comment]
> > Is that correct that when you state only `netsuite.fetchTarget.types.[type_name]` you don't **all** fetch the files as well?
> > It's written in that if you don't provide `netsuite.fetchTarget.filePaths` it fetches all file cabinet content, while in the examples it seems like it doesn't.
> > I think that some explanation is missing.
> 
> That's true. The default of `types` and` file Paths` is to fetch nothing, but the default of the whole `fetchTarget` is to fetch everything. Might be a bit trickier to explain.

Removed the 'default' table column and added an explanation about the default above. Let me know what you think

File: packages/netsuite-adapter/specific-cli-options.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# NetSuite Specific CLI options
+## Fetch Target
+Salto CLI allows the user to fetch only specific netsuite configuration elements and/or file cabinet items by appending the ```netsuite.fetchTarget.filePaths``` and ```netsuite.fetchTarget.types.[type_name]``` parameters following the ```-C``` CLI option.
+When none of the ```fetchTarget``` parameters is specified, Salto will fetch all configuration elements.
+
+| Name                           |  Description
+| -------------------------------| ------------------------| -----------
+| netsuite.fetchTarget.filePaths | A list of regular expressions of cabinet file paths. Only files with matching paths will be fetched.
+| netsuite.fetchTarget.types.[type_name]  | A list of script id regular. Only records of type ```type_name``` with matching script ids will be fetched.
```
Comment (alonstern) [inline packages/netsuite-adapter/specific-cli-options.md:9]:
```suggestion
| netsuite.fetchTarget.types.[type_name]  | A list of script id regular expressions. Only records of type ```type_name``` with matching script ids will be fetched. 
```

**Review by omrilit — APPROVED** [review]
I think it's much clearer now!
Maybe you can consider adding one example of using both `filePaths` and `types` options in the same command

File: packages/netsuite-adapter/specific-cli-options.md (hunk 2 above)
Comment (tomermevorach) [reply packages/netsuite-adapter/specific-cli-options.md]: 10x fixed

**Pull request merged**